pkg/apis/config/register.go

```diff
@@ -47,4 +48,5 @@ func init() {
 	// generated functions takes place in the generated files. The separation
 	// makes the code compile even when the generated files are missing.
 	localSchemeBuilder.Register(addKnownTypes)
```

Comment:
Keep the initial case of words in the code consistent

Reply:
This is automatically generated code. The case of method names controls visibility.